ILRepack.Tests/ILRepack.Tests.csproj

```diff
@@ -55,6 +58,15 @@
     <None Include="packages.config">
       <SubType>Designer</SubType>
     </None>
+    <None Include="sharpdevelop\ICSharpCode.SharpDevelop.BuildWorker120.exe.config" />
```

Comment:
You're right, it should be a lot easier to get them from nuget. But we need a project like sharpdevelop, which has quite a lot of dlls. I thought that nuget is more for standalone libraries. Will investigate more.